containers/src/Kokkos_UnorderedMap.hpp

```diff
@@ -393,9 +394,9 @@ class UnorderedMap {
   ///
   /// This method has undefined behavior when erasable() is true.
   ///
-  /// Note that this is not a device function; it cannot be called in
+  /// Note that this is <i>not</i> a device function; it cannot be called in
```

Comment:
```suggestion
  /// Note that this is *not* a device function; it cannot be called in
```
maybe rather markdown?

Reply:
I was just following what was already done in the rest of the file.

I'd rather stay consistent here.  If you want me to change it (I'm generally not a fan of markdown in comments either), I'd rather do it under a separate P/R.

Reply:
Oh, sure. I wasn't aware of the style of comments in the rest of the file.